src/main/java/uk/gov/hmcts/reform/sscs/service/evidence/EvidenceUploadService.java

```diff
@@ -122,7 +123,11 @@ private String getCreatedDate(Document document) {
                     documentExtract.setDocuments().accept(caseDetails.getData(), newDocuments);
 
                     ccdService.updateCase(caseDetails.getData(), caseDetails.getId(), eventType.getCcdType(), summary, UPDATED_SSCS, idamService.getIdamTokens());
-
+                    try {
+                        log.info("uploadEvidence: for case ID Case({}): User has uploaded the file ({}) with a checksum of ({})", caseDetails.getId(), file.getOriginalFilename(), DigestUtils.md5DigestAsHex(file.getBytes()).toUpperCase());
+                    } catch (IOException ioException) {
+                        log.error(ioException.getMessage() + ". Something has gone wrong for caseId: ", caseDetails.getId() + " when logging uploadEvidence");
```

Comment:
can we add checksum into this log msg pls also

Reply:
Done, pls check again